src/main/resources/META-INF/plugin.xml

```diff
@@ -2,6 +2,13 @@
     <id>com.redhat.devtools.intellij.kubernetes</id>
     <name>Kubernetes by Red Hat</name>
     <vendor email="developers@redhat.com" url="http://www.redhat.com">Red-Hat</vendor>
+    <description>
+        <![CDATA[
+        <h2>Overview</h2>
+        <p>A plugin for interacting with Kubernetes and OpenShift clusters.</p>
+        <p>The plugin provides functionalities and user experiences that are very close to the <a href="Kubernetes extension for vscode">https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools</a>.</p>
```

Comment:
I think this must be removed because it's very likely JetBrains will refuse it.

Reply:
What is striking is that **"Overview"** is repeated as such when you search:
![image](https://user-images.githubusercontent.com/25126/105989499-827a8200-60a1-11eb-8cdb-ce84fe7d1770.png)
Then on the plugin page itself, "Overview" seems repeating what the page already states:
![image](https://user-images.githubusercontent.com/25126/105989822-f0bf4480-60a1-11eb-9fba-7339ee31fb63.png)
I'd thus remove "Overview" and drop the **icon** that also repeats the icon that's already displayed on top.

HTML per se seems supported as **links** that we have on other plugin pages are displayed:
![image](https://user-images.githubusercontent.com/25126/105990041-3976fd80-60a2-11eb-88a5-41b563970b5c.png)

Do you want me to remove the link to the vscode extension, is it what you mean?
 

Shouldn't we indicate a description in the plugin.xml?